tests/llm/remote_evals/__main__.py

```
@@ -0,0 +1,81 @@
+        test_cases = get_test_cases(TEST_CASES_FOLDER)
+
+        # Setup all infrastructure upfront
+        try:
+            await infrastructure_manager.setup_all_once(test_cases)
+        except Exception as e:
+            print(f"❌ Failed to setup infrastructure: {e}")
+            print("⚠️  Server startup aborted")
+            sys.exit(1)
+
+        # Register cleanup on shutdown
+        def cleanup_handler(signum, frame):
```

Ruff (0.14.10)

Unused function argument: `signum`

(ARG001)